addon/templates/components/html-select.hbs

```diff
@@ -1,4 +1,4 @@
-<select {{action 'change' on='change'}} class="form-control">
+<select {{action 'change' on='change'}} class="{{mainComponent.elementClass}}">
```

Comment:
form-control is the default value of elementClass, there are cases where I do not want to use form-control on this element.